File: ART_MD_C/ch02_2/InitCoords.c
void InitCoords ()
{
  VecR c, gap;
  int n, nx, ny;

  VDiv (gap, region, initUcell);
  n = 0;
  for (ny = 0; ny < initUcell.y; ny ++) {
    for (nx = 0; nx < initUcell.x; nx ++) {
      VSet (c, nx + 0.5, ny + 0.5);
      VMul (c, c, gap);
      VVSAdd (c, -0.5, region);
      mol[n].r = c;
      ++ n;
    }
  }
}

File: ART_MD_JL/ch02_case02/init_coords.jl
function init_coords!(
    atoms::Atoms,
    inp::InputVars,
    region
)
    initUcell = inp.initUcell
    gap = region ./ initUcell
    c = [0.0, 0.0]
    ia = 0
    for ny in 0:initUcell[2] - 1
        for nx in 0:initUcell[1] - 1
            ia = ia + 1
            c[1] = (nx + 0.5)*gap[1]
            c[2] = (ny + 0.5)*gap[2]
            # Set positions
            atoms.r[1,ia] = c[1] - 0.5*region[1]
            atoms.r[2,ia] = c[2] - 0.5*region[2]
        end
    end
    return
end
